quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -29,6 +31,8 @@ public final class LoggingPropertyMappers {
     private static final String SYSLOG_ENABLED_MSG = "Syslog is activated";
     private static final String DEFAULT_ROOT_LOG_LEVEL = toLevel(LoggingOptions.LOG_LEVEL.getDefaultValue().orElseThrow().get(0)).getName();
 
+    private static List<CategoryLevel> rootLogLevels;
```

Comment:
Just to be future proof for testing, can this be reset by AbstractConfigurationTest.resetConfiguration?

Reply:
@shawkins Thanks for spotting that.

I rather have it isolated and reset in the prop mappers class below. Should be ok now. 

https://github.com/keycloak/keycloak/pull/37265/files#diff-78e1ea4992f437dbda8637131776d9f1e448a2cf4f6ba2dd986e07ec0d1eb298R40

Reply:
Sure, that's fine too as it's called by a reset method already. 

Maybe someday we'll have some additional context for static state, rather than having it spread about :)